### ISSUE TITLE
Connect to Mongodb from serveless backend

### ISSUE THREAD

**Issue by davisokoth** [Coding-Coach/coding-coach-api#42]
Data will be stored on an Atlas-hosted Mongodb instance. The api will connect to it to store and retrieve details

**Comment by moshfeu** [comment]
We probably need to sketch the database for all the needs (collections and so).
The first of us who will start, will share with the others the document.

**Comment by crysfel** [comment]
Agreed with @moshfeu, but for now I will take this as a just setting up the Azure stack, then we can go ahead and define all collections, fields and everything we need.

**Comment by moshfeu** [comment]
Sounds good!
